pkgs/development/python-modules/gpib-ctypes/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, pip
+, bumpversion
+, wheel
+, watchdog
+}:
+
+buildPythonPackage rec {
+  version = "0.3.0";
+  pname = "gpib_ctypes";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1blivb1zy2knf90d3n44jwy5a5rmnz122vgaqjwg11mmav319r8g";
+  };
+
+  checkPhase = ''
+    py.test
+  '';
+
+  checkInputs = [ pytest ];
+
+  nativeBuildInputs = [ pip bumpversion wheel watchdog ];
+
+  meta = with lib; {
+    description = "Cross-platform Python bindings for the NI GPIB and linux-gpib C interfaces";
+    homepage = https://github.com/tivek/gpib_ctypes/;
```

Comment:
this needs to be a string, when trying to build this derivation

```
 error: URL literals are disabled

       at /home/kirillvr/.cache/nixpkgs-review/pr-231989-3/nixpkgs/pkgs/development/python-modules/gpib-ctypes/default.nix:30:16:

           29|     description = "Cross-platform Python bindings for the NI GPIB and linux-gpib C interfaces";
           30|     homepage = https://github.com/tivek/gpib_ctypes/;
             |                ^
           31|     license = licenses.gpl2Only;

```

Reply:
I missed this part. Thanks for noticing. Already updated it.